lib/compiler-llvm/Cargo.toml

```diff
@@ -20,15 +20,15 @@ wasmer-compiler = { path = "../compiler", version = "=6.0.1", features = [
 ] }
 wasmer-vm = { path = "../vm", version = "=6.0.1" }
 wasmer-types = { path = "../types", version = "=6.0.1" }
-target-lexicon = { version = "0.12.2", default-features = false }
-smallvec = "1.6"
+target-lexicon = { workspace = true, default-features = false }
+smallvec.workspace = true
 object = { workspace = true, features = ["read"] }
 libc.workspace = true
-byteorder = "1"
-itertools = "0.10"
-rayon = "1.5"
+byteorder.workspace = true
+itertools.workspace = true
+rayon.workspace = true
 phf = { version = "0.11.2", features = ["macros"] }
-tracing = { version = "0.1", default-features = false, features = ["log"] }
+tracing = { workspace = true, features = ["log"] }
```

Comment:
```suggestion
tracing.workspace = true
```

Reply:
We register a tracing subscriber, so according to the docs, the `log` feature should have no effect anyway. But I am not sure what our subscribers do, maybe they are accessing log data.

We probably don't want the `log` feature of tracing enabled only when we add the llvm compiler as a dependency. So this is probably not the right place to set it, and it should be removed, or moved to the workspace, or the cli Cargo.toml

Reply:
I don't disagree, but this PR is keeping the current behavior of the codebase.
If we want to change the behavior that should be done in a PR apart